### PR TITLE
[Proposal] Show list of available translations

### PR DESCRIPTION
**Pull request by Zebradil**
Hi!

I have an idea to show list of available translations somewhere. Currently I placed the list right under post title. It displays if there is at least one translation for the post. In example below I created post in three languages: English, Spanish and Russian.

For English version:
screen shot 2018-04-03 at 23 51 23

For Russian version:
screen shot 2018-04-03 at 23 52 39

What do you think?

### CODE REVIEW

File: layouts/post/single.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
       <h1 class="post-title">{{ .Title }}</h1>
 
       <div class="post-meta">
+        <div class="post-meta">
```
Comment (xianmin) [inline layouts/post/single.html]: Maybe, it's better to palce an `{{ if .IsTranslated }}   {{ end }}` between these code.

Or, only modify the `i18nlist.html` , like this:

```
{{ if .IsTranslated }}  
<div class="post-meta">
  ......
</div>
{{ end }}
```

**Comment by xianmin** [comment]
This proposal is fantastic!  :smile: 
I commented on a small piece of code.

File: layouts/post/single.html (hunk 1 above)
Comment (Zebradil) [reply layouts/post/single.html]: Actually, I took the code from the Hugo docs and initially there was this condition `{{ if .IsTranslated }} `. But in this case I don't see translations in post version written in default language. That's why I removed it.

That's strange — now it is working with this condition. I'll update the PR.

**Comment by Zebradil** [comment]
I'm not sure how to do it nice in meaning of design. Maybe it's a good idea to underline links as it's done with post title? I also thought about adding flags to emphasize these links (🇬🇧🇷🇺🇪🇸🇨🇳).

**Comment by xianmin** [comment]
Thanks for your Pull Requests! Maybe this is enough for now.
I'm not sure how many people will need this feature.
If they have better proposal, we can improve. :smile:

**Comment by Zebradil** [comment]
Great, thanks!

**Pull request merged**